testing/README.md

```diff
@@ -2,26 +2,27 @@
 
 ## Philosophy
 * Test code is a first-class part of the deliverable, not an optional nice-to-have.
-* When scopeing / estimating work: always factor in testing. 
-* Untestable code = bad code.
+* When scoping / estimating work: always factor in testing. 
 * Insufficent test coverage is better than no test coverage.
+* Prefer [Use Case Coverage over Code Coverage](https://kentcdodds.com/blog/how-to-know-what-to-test) (no need for 100% code coverage).
 * Additional history and guiding principles are in [this "manitesto" doc](https://bitbucket.org/blog/save-time-with-default-pull-request-descriptions)
 
 ## Best Practices & Standardization
 
 ### General
 * Organization of `scripts` in package.json:
-	- `test` = runs all unit tests + E2E tests in headless mode
+	- `test` = runs all unit/integration/E2E tests in headless mode
 	- `test:watch` = same as `test` but with Jest in watch mode
-	- `cypress` = run cypress with the dashboard open
+	- `cypress` = run Cypress with the [Cypress Test Runner](https://docs.cypress.io/guides/guides/command-line.html#cypress-open)
 	- `check-lint` = runs style lint, eslint, prettier
 	- `lint` = same as `check-lint` but writes the changes (`--fix`)
 	- `ci` = run EVERYTHING
 * Try to minimize nesting and coupling. These make tests brittle and hard to maintain.
+* [Apply the AHA principle (try to avoid over-abstraction)](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing).
```

Comment:
I have also found applying the AHA (Avoid Hasty Abstractions) principle to be a great way to approach writing tests. It's led to less cognitive overhead when writing, reading, and refactoring any previous tests I've written. 👍

Reply:
This seems a bit redundant with the proceeding point. Is there a way to merge them?

Reply:
Thanks, @emersonthis, I like your suggestion! I'll have to come back and think of an alternative that merges the two. 😉

Reply:
@emersonthis Does this work better? I'm open to suggestions on rewording it if you've got something better. 🙂 

https://github.com/cloudfour/guides/pull/68/commits/56ebd0842154ca1e57f02528203b82b13559cbfd